packages/suite/src/services/suite/metadata/GoogleProvider.ts

```diff
@@ -96,6 +96,28 @@ class GoogleProvider extends AbstractMetadataProvider {
         }
     }
 
+    async getMetadataFilesList() {
+        try {
+            const response = await GoogleClient.list({
+                query: { spaces: 'appDataFolder' },
+            });
+
+            if (response) {
+                const formattedList = response.files.map(({ name }) => name);
+
+                return this.ok(formattedList);
+            }
+
+            return this.ok(undefined);
+        } catch (error) {
+            if (error?.error?.code === 404) {
```

Comment:
404 is return when the provider is empty? just trying to  validate that 404 is really "ok" here.

Reply:
Actually I believe it should never even be 404 in here, since the `appDataFolder` definitely exists after the provider had been initialized, right?